pycolmap/scene/camera.h

```diff
@@ -184,24 +200,11 @@ void BindCamera(py::module& m) {
       .def("__copy__", [](const Camera& self) { return Camera(self); })
       .def("__deepcopy__",
            [](const Camera& self, py::dict) { return Camera(self); })
-      .def("__repr__", [](const Camera& self) { return PrintCamera(self); })
-      .def("summary", [](const Camera& self) {
```

Comment:
It is a bit annoying to have to look up which classes have `.summary`. Maybe add it to all classes, and by default just redirect to `__repr__`? `__repr__` for a very short string identifier, `.summary` for a longer, more detailed description.

Reply:
Now we generate the summary automatically.